pkcs11mechanism.c

```diff
@@ -81,16 +92,31 @@ ZEND_END_ARG_INFO()
 
 PHP_METHOD(Mechanism, __debugInfo) {
 
-  const pkcs11_mechanism_object * const o = Z_PKCS11_MECHANISM_P(ZEND_THIS);
+    const pkcs11_mechanism_object * const o = Z_PKCS11_MECHANISM_P(ZEND_THIS);
 
-  ZEND_PARSE_PARAMETERS_NONE();
+    ZEND_PARSE_PARAMETERS_NONE();
```

Comment:
(minor) please avoid changing the indentation along with another purpose of your commit, even if it is a valid change.

Reply:
Yeah, I even remind my own team of this at work. Cordonnier mal chaussé.